client/app/services/parameters/Parameter.js

```diff
@@ -146,6 +151,7 @@ class Parameter {
     return `{{ ${this.name} }}`;
   }
 
+  /** Get a saveable version of the Parameter by omitting unnecessary props */
```

Comment:
@ranbena I'm thinking about using JSDoc syntax for the most confusing methods (can either be as simple as that or hold more info). Considering #4135 it would be helpful in VSCode for autocomplete :)